presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcColumnHandle.java

```diff
@@ -70,9 +74,15 @@ public boolean isNullable()
         return nullable;
     }
 
+    @JsonProperty
+    public Optional<String> getComment()
+    {
+        return comment;
+    }
+
     public ColumnMetadata getColumnMetadata()
     {
-        return new ColumnMetadata(columnName, columnType, nullable, null, null, false, emptyMap());
+        return new ColumnMetadata(columnName, columnType, nullable, comment.orElse(null), null, false, emptyMap());
```

Comment:
While here, could you please annotate `io.prestosql.spi.connector.ColumnMetadata#getComment` as `@Nullable`? and other nullable stuff there too?